webpack.builtinblocks.config.js

```diff
@@ -79,10 +79,6 @@ module.exports = {
             './xmodule/js/src/xmodule.js',
             './xmodule/js/src/sequence/edit.js'
         ],
-        VideoBlockDisplay: [
-            './xmodule/js/src/xmodule.js',
-            './xmodule/js/src/video/10_main.js'
-        ],
```

Comment:
I don't see a reason to move this to webpack.common.config.js or to change the name of the entrypoint. After all, Video is still a built-in block (for now). Would it be work to change this to:
```js
        VideoBlockDisplay: [
            './xmodule/js/src/video/10_main.js'
        ],
```

Reply:
Yes, its working
I was actually trying to make this block configs similar to vertical block as it was mentioned as an example one in the readme.md [here](https://github.com/openedx/edx-platform/blob/master/xmodule/assets/README.rst#javascript-js)
but I think it make more sense as you are suggesting.